src/Microsoft.Graph.Core/Requests/GraphClientFactory.cs

```diff
@@ -56,33 +51,25 @@ public static class GraphClientFactory
         /// Germany endpoint
         public const string Germany_Cloud = "Germany";
 
-        /// <summary>
-        /// Proxy to be used with created clients
-        /// </summary>
-        public static IWebProxy Proxy { get; set; }
-
-        /// <summary>
-        /// DefaultHandler is a Func that returns the HttpMessageHandler for actually making the HTTP calls.
-        /// The default implementation returns a new instance of HttpClientHandler for each HttpClient.
-        /// </summary>
-        public static Func<HttpMessageHandler> DefaultHttpHandler = () => {
-            return new HttpClientHandler
-            {
-                Proxy = Proxy
-            };
-        };
-
         /// <summary>
         /// Creates a new <see cref="HttpClient"/> instance configured with the handlers provided.
         /// </summary>
         /// <param name="authenticationProvider">The <see cref="IAuthenticationProvider"/> to authenticate requests.</param>
         /// <param name="version">The graph version to use.</param>
         /// <param name="nationalCloud">The national cloud endpoint to use.</param>
+        /// <param name="proxy">The proxy to be used with created client.</param>
+        /// <param name="innerHandler">The last HttpMessageHandler to HTTP calls.
+        /// The default implementation creates a new instance of <see cref="HttpClientHandler"/> for each HttpClient.</param>
         /// <returns></returns>
-        public static HttpClient Create(IAuthenticationProvider authenticationProvider, string version = "v1.0", string nationalCloud = Global_Cloud)
+        public static HttpClient Create(
+            IAuthenticationProvider authenticationProvider,
+            string version = "v1.0",
+            string nationalCloud = Global_Cloud,
+            IWebProxy proxy = null,
+            HttpMessageHandler innerHandler = null)
```

Comment:
Perhaps call this finalHandler to suggest that is what is supposed to be used here.